data/org.mate.pluma.gschema.xml.in

```diff
@@ -50,6 +50,11 @@
       <summary>Autosave Interval</summary>
       <description>Number of minutes after which pluma will automatically save modified files.  This will only take effect if the "Autosave" option is turned on.</description>
     </key>
+    <key name="hide-trailing-newline" type="b">
+      <default>true</default>
+      <summary>Hide Tailing Newline</summary>
+      <description>Whether pluma should hide the final newline in opened files if one was present when loading the file.  Enabling this will also cause newly created files to be saved with a trailing newline by default.</description>
```

Comment:
This seems wrong. The fact that "hiding" the newline causes the newline to be added is unexpected behaviour.